addons/l10n_no/views/res_partner_views.xml

```diff
@@ -5,7 +5,7 @@
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='vat']" position="after">
+            <xpath expr="//group/group" position="inside">
```

Comment:
this XPATH is maybe a little too much generic, if there was a customization adding a group in first, this field will now be in another group

I think we had the issue for other things and it was caused by the field vat being overridden by:

https://github.com/odoo/odoo/blob/20b2d6abd864423a89a18ff5f0ad47fda2a7f1de/addons/base_vat/views/res_partner_views.xml#L10-L16

so the `l10n_no_bronnoysund_number` is added inside a div associated to "Tax Id" label, instead of being added after "Tax Id" field as a whole.

I don't remember what we did for the other case, but good solution would be:

1) have a higher priority to the view (the `base_vat.view_partner_base_vat_form` has priority 15, so for example 14 would work (currently, this view has the default value which is 16 and the lowest priority is applied first))

2)  something like your current fix (but there is the small issue that I spoke of)

3) do a more complex XPATH

the three ways are ok to me